src/waveform/renderers/waveformmarkset.cpp

```diff
@@ -18,10 +17,8 @@ void WaveformMarkSet::setup(const QString& group, const QDomNode& node,
                             const SkinContext& context,
                             const WaveformSignalColors& signalColors) {
 
-
-#if QT_VERSION >= 0x040700
-    m_marks.reserve(NUM_HOT_CUES);
-#endif
+    m_marks.reserve(NUM_HOT_CUES + 3);
```

Comment:
could you add a comment about the + 3? (it's one for the load cue and one for the in/out cues, right?)